github/workflows/rules/md101.js

```diff
@@ -0,0 +1,91 @@
+const { InlineTokenChildren } = require("./common/inlineTokenChildren");
+const { WordPattern } = require("./common/wordPattern");
+
+const keywords = [
+    new WordPattern("iExtractor-manager"),
```

Comment:
This is specific to Malus Security. It does no harm to us, but do we need it?

Reply:
@davidxbors , remove it.

Reply:
Yes, it's supposed to be a template. I removed `iExtractor-manager` and other Malus Security specific keywords.